k8s/cmd/commands/util/certmanager.go

```diff
@@ -45,3 +57,20 @@ func GetCertManagerManifests(certManagerManifestURL string) (Manifests, error) {
 	}
 	return ParseManifestsToObjects(manifests)
 }
+
+func GetCertManager() (Manifests, error) {
+	var certManagerManifests Manifests
+	var err error
+	if flags.CertManagerVersion == flags.DefaultCertManagerVersion {
+		certManagerManifests, err = getCertManagerManifestsFromLocal()
+		if err != nil {
+			log.Fatal(err, "failed to get cert-manager manifests from local")
+		}
+	} else {
+		certManagerManifests, err = getCertManagerManifestsFromURL(getCertManagerURL())
+		if err != nil {
+			log.Fatal(err, "failed to get cert-manager manifests from remote")
+		}
```

Comment:
Use the bundled cert-manager directly. I doesn't make sense to use a user-specific, _untested_ version of cert-manager for vineyard-operator, to simplify the process.

You can just remove the check, and remember to update the corresponding documentation and command line arguments as well.

Reply:
Make sense to me.